lib/cea/cea608_memory.js

```diff
@@ -80,121 +80,10 @@ shaka.cea.Cea608Memory = class {
    */
   forceEmit(startTime, endTime) {
     const stream = `CC${(this.fieldNum_<< 1) | this.channelNum_ +1}`;
-
```

Comment:
I've broken this logic out into a utility, since the same logic is needed for CEA-708 caption emission.